src/ansys/motorcad/core/geometry.py

```diff
@@ -515,6 +515,140 @@ def edit_point(self, old_coordinates, new_coordinates):
                 # Check Arc is still valid
                 _ = entity.centre
 
+    def round_corner(self, corner_coordinate, radius):
+        """Round the corner of a region.
+
+        The corner coordinates must already exist on two entities belonging to the region.
+        The two entities adjacent to the corner are shortened, and an arc is created between
+        them.
+
+        Parameters
+        ----------
+        corner_coordinate : ansys.motorcad.core.geometry.Coordinate
+            Coordinate of the corner to be rounded.
+        radius : float
+            Radius by which the corner will be rounded.
```

Comment:
```suggestion
        corner_coordinate : ansys.motorcad.core.geometry.Coordinate
            Coordinate of the corner to round.
        radius : float
            Radius to round the corner by.
```
Use present tense rather than future tense verbs.